packages/x-components/src/design-system/components/suggestion/default.scss

```
@@ -74,4 +74,42 @@
   > .x-identifier-result {
     flex: none;
   }
+
+  &.x-suggestion--is-curated {
```

```suggestion
  &.x-suggestion.x-suggestion--is-curated {
```

Do it like we do in tags to give them more specificity